src/Controller/SkillController.php

```diff
@@ -17,28 +23,50 @@ public function index(Request $request, Session $session)
     {
         //        Recherche des activites
 
-        $a = new \activites();
-        $a->fetch();
-        $activites = $a->elements;
+        $query = $this->entityManager->getRepository(Skill::class)->findBy(array('supprime' => NULL));
```

Comment:
remplacer
<pre>$query = $this->entityManager->getRepository(Skill::class)->findBy(array('supprime' => NULL));</pre>
par
<pre>$activites = $this->entityManager->getRepository(Skill::class)->findBy(array('supprime' => NULL));</pre>

Et supprimer les lignes 27 à 33 ($activites = array(); ... If ... foreach {}}